client/web/compose/src/components/PageBlocks/ChartConfigurator.vue

```diff
@@ -43,16 +44,31 @@
           class="mb-2"
         />
 
-        <c-input-select
-          v-model="options.drillDown.blockID"
-          :options="drillDownOptions"
-          :get-option-key="getOptionKey"
-          :disabled="!options.drillDown.enabled"
-          :get-option-label="o => o.title || o.kind"
-          :reduce="option => option.blockID"
-          :clearable="true"
-          :placeholder="$t('chart.drillDown.openInModal')"
-        />
+        <b-input-group>
+          <c-input-select
+            v-model="options.drillDown.blockID"
+            :options="drillDownOptions"
+            :get-option-key="getOptionKey"
+            :disabled="!options.drillDown.enabled"
+            :get-option-label="o => o.title || o.kind"
+            :reduce="option => option.blockID"
+            :clearable="true"
+            :placeholder="$t('chart.drillDown.openInModal')"
+          />
+
+          <b-input-group-append>
+            <column-picker
+              ref="columnPicker"
+              :module="selectedChartModule"
+              :fields="selectedFields"
```

Comment:
when column picker is used in a b-input-group, the button variant should be 'extra-light'